mopidy_mpd/dispatcher.py

```diff
@@ -239,6 +239,9 @@ class MpdContext:
     #: The subsytems that we want to be notified about in idle mode.
     subscriptions = None
 
+    #: Whether to use multiple tags for e.g. artists.
+    multiple_tags = None
```

Comment:
I think the default value of this property should be `False`, not `None`.

Reply:
I’m not sure; a default value is set in `ext.conf`. But `MpdContext.__init__` checks whether `config is not None`, so it doesn’t require a configuration. Are there cases in which there is no configuration passed to `MpdContext.__init__` but this value *is* required?

Reply:
Each class should be usable by itself without knowledge of the system as a whole. This property is designed to hold booleans, so it should never be None or anything else.

Reply:
Fair enough.